rednotebook/util/markup.py

```diff
@@ -265,6 +269,17 @@ def _get_config(target, options):
         config['preproc'].append([r'\\\(\s*(.+?)\s*\\\)', r"BEGINMATH''\1''ENDMATH"])
         config['postproc'].append([r'BEGINMATH(.+)ENDMATH', r'$\1$'])
 
+        # Fix utf8 quotations - „, “ and ” cause problems compiling the latex document
+        config['postproc'].append([u'„', '"'])
+        config['postproc'].append([u'”', '"'])
+        config['postproc'].append([u'“', '"'])
```

Comment:
That is true. But if you copy a lot from the web, e.g. news articles, you get a ton of this three symbols. If you also change the encoding to `utf8x`, there are only a few symbols left to cause problems.